config/locales/en.yml

```diff
@@ -310,6 +324,16 @@ en:
             one: "A new job matching your search criteria was posted yesterday."
             other: "%{count} new jobs matching your search criteria were posted yesterday."
           unsubscribe: Follow this link to unsubscribe.
+    expiry:
+      email:
+        resubscribe: Resubscribe now
+        first_warning:
+          subject: "Job alert for '%{reference}' expiring - resubscribe now"
+          first_line: "Your job alert for your alert '%{reference}' on Teaching Vacancies will expire on %{expiry_date}. After this date you will no longer receive emails alerting you to new jobs that match these criteria."
```

Comment:
Extra `your alert` in here, I think. Should be `Your job alert for '%{reference}'`.